package.json

```diff
@@ -6,7 +6,7 @@
   "author": "",
   "license": "MIT",
   "scripts": {
-    "lint": "eslint 'packages/*/src/**/*.ts' --quiet --fix",
+    "lint": "eslint 'packages/*/src/**/*.ts' --quiet --fix --report-unused-disable-directives",
```

Comment:
Put this in the eslintrc.js

Reply:
https://eslint.org/docs/user-guide/configuring#report-unused-eslint-disable-comments

Reply:
This only add to warnings; not failure. Keeping command line option